pandas/tests/test_downstream.py

```diff
@@ -136,7 +136,12 @@ def test_missing_required_dependency():
     # https://github.com/MacPython/pandas-wheels/pull/50
     call = ["python", "-sSE", "-c", "import pandas"]
 
-    with pytest.raises(subprocess.CalledProcessError) as exc:
+    msg = (
+        r"Command '\['python', '-sSE', '-c', 'import pandas'\]' "
+        r"returned non-zero exit status 1."
```

Comment:
```suggestion
        "returned non-zero exit status 1."
```

I don't think need the raw prefix here either